project.yml

```
@@ -100,3 +100,6 @@ targets:
       - path: MapboxTestHost
     dependencies:
       - package: MapboxMaps
+        product: MapboxMaps
+      - package: MapboxMaps
+        product: CocoaImageHashing
```

Shouldn't this just be…

```suggestion
      - package: CocoaImageHashing
```

---

It doesn't like that. We get `Spec validation error: Target "MapboxTestHost" has an invalid package dependency "CocoaImageHashing"`

---

Ah, you also need to add an entry to the top-level packages key. See the xcodegen docs for more info and feel free to reach out to me if you want to walk through it.